tdrs-backend/tdpservice/core/views.py

```diff
@@ -13,12 +12,6 @@
 logger = logging.getLogger()
 
 
-class IndexView(TemplateView):
```

Comment:
The template for this was removed, but the URL route and view remained - causing a 500 error when navigating to the root URL.